### ISSUE TITLE
Darstellung der RSSI Linie

### ISSUE THREAD

**Issue by friedpa** [jp112sdl/AskSinAnalyzer#60]
Ich habe eine Frage zu der Darstellung der roten RSSI Linie. Wenn ich den ASA einschalte, zeichnet er mir die RSSI Linie auf, aber nur solange bis der Zoom von 1m auf 5m umspringt. Dann ist es vorbei mit der roten Linie :o) Das ganze passiert auf einem Firefox Browser (letzte Version) und einem IE.

![grafik](https://user-images.githubusercontent.com/42890695/78903311-3a8b5f80-7a7b-11ea-90ed-c85fc2b7ae82.png)

Liebe Grüße
Paul

**Comment by psi-4ward** [comment]
Die sollte natürlich weiter laufen. Hast du den Chrome probiert? Steht was in der developer console (F12)?

**Comment by friedpa** [comment]
Chrome habe ich keinen installiert .... zu resourcenintensiv.
bei der Developer Console musst Du mir helfen. Von welchem Reiter soll ich einen Screenshot machen?
Hier einmal der Inspektor:

![grafik](https://user-images.githubusercontent.com/42890695/78907022-69580480-7a80-11ea-9445-4d7802397e81.png)

**Comment by psi-4ward** [comment]
Einfach mal im Reiter "Konsole" kucken ob was rot wird.

**Comment by friedpa** [comment]
Ergebnis:

![grafik](https://user-images.githubusercontent.com/42890695/78917126-bf33a900-7a8e-11ea-935b-b62dfaa446e2.png)

**Comment by psi-4ward** [comment]
Das sagt mir ja mal gar nichts. Wenn du es auf 1min lässt läuft es aber als weiter?

**Comment by friedpa** [comment]
Nope, sobald der 5m Tab oben links auf aktiv geht ist es vorbei mit der roten Linie.

**Comment by psi-4ward** [comment]
Ich kann das leider nicht nachstellen. 
Welche Version vom Firefox nutzt du? ESP und AVR Software ist aktuell?

**Comment by psi-4ward** [comment]
Wie verhält es sich wenn du nach ca 15 Minuten die WebUI neu aufrufst?

**Comment by friedpa** [comment]
Zur Frage 1: Firefox 75.0 (64-Bit)
Frage 2: Firefox stellt die rote Linie wieder dar:
![grafik](https://user-images.githubusercontent.com/42890695/78970462-a74d3b00-7b09-11ea-842f-70a0650e14fb.png)
Der Fehler tritt nur beim erstmaligen "befüllen" des ASA auf

**Comment by psi-4ward** [comment]
Das Problem lässt sich nicht nachstellen. Hast du die aktuellen Versionen auf dem avr und esp?

**Comment by jp112sdl** [comment]
Habe selben Browser (auf macOS) verwendet.
Meine RSSI Linie bleibt dauerhaft erhalten.

_Habe es jedoch nicht über mehrere Stunden getestet_

**Comment by friedpa** [comment]
Ich habe die aktuellen Versionen:

![grafik](https://user-images.githubusercontent.com/42890695/79323535-9ee07000-7f0e-11ea-9137-b8f01399f672.png)

Wenn sonst niemand das Problem hat wird es wohl an meinem Setup liegen. Wie schon gesgt, die rote Linie wird nicht mehr dargestellt wenn der 5m Button oben links aktiv wird.

**Comment by jp112sdl** [comment]
> wenn der 5m Button oben links aktiv wird.

<img width="1037" alt="Bildschirmfoto 2020-04-15 um 11 52 57" src="https://user-images.githubusercontent.com/18190139/79324312-b79d5580-7f0f-11ea-8f04-1e2c57484e06.png">

**Comment by friedpa** [comment]
@jp112sdl Warum ist die rote Linie bei Dir nicht über die ganze Breite des Diagrams?

**Comment by jp112sdl** [comment]
Das ist by Design.
Der Analyzer hält die letzten 1000 Telegramme vor, sowie die letzten 255 RSSI Messungen.

Da RSSI alle 750ms übertragen wird, hat man beim **Neu-Aufruf** der WebUI nur rund 3 Minuten (255 * 0,75 Sekunden) rückwirkend die RSSI Linie.

Heißt in meinem Bild: Ich habe die WebUI so ca. 11:49:45 Uhr geladen.

**Comment by friedpa** [comment]
Darf ich hiezu noch eine Frage stellen: Warum habe ich eine rote Linie von 14:32h bis 15:37h also 65 Minuten und dann stoppt diese?

![grafik](https://user-images.githubusercontent.com/42890695/79344164-bda32e80-7f2f-11ea-8852-245a6e64e517.png)

**Comment by jp112sdl** [comment]
Gibt es derzeit noch Probleme mit der RSSI Linie oder kann das Ticket zu?

**Comment by friedpa** [comment]
Hat sich nichts getan, Fehler ist immer noch da:

![grafik](https://user-images.githubusercontent.com/42890695/84866948-b953d880-b07a-11ea-89b0-332cb09794ec.png)

Verwende als Browser Firefox

**Comment by jp112sdl** [comment]
Komisch, dass das nur bei dir so ist :/
